aws_lambda_powertools/utilities/batch/exceptions.py

```diff
@@ -38,6 +38,15 @@ def __str__(self):
         return self.format_exceptions(parent_exception_str)
 
 
+class UnexpectedBatchTypeError(BatchProcessingError):
+    """Error thrown by the Batch Processing utility when a partial processor receives an unexpected batch type"""
+
+    def __init__(self):
+        msg = "Unexpected batch type. Possible values are: SQS, KinesisDataStreams, DynamoDBStreams"
+        super().__init__(msg)
+        self.name = "UnexpectedBatchTypeError"
+
+
```

Comment:
```suggestion
class UnexpectedBatchTypeError(Exception):
    """Error thrown by the Batch Processing utility when a partial processor receives an unexpected batch type"""
    
    pass
```

This is a simple exception where you don't need to handle messages and stuff like that, so you can inherit from Exception and delegate the message/info/raising to whoever is calling this exception.

Reply:
Got it, thanks! Changed it